base_classes/nyaml/NXdata.yaml

```diff
@@ -237,6 +237,21 @@ NXdata(NXobject):
       .. note::  Attributes potentially containing multiple values
          (axes and _indices) are to be written as string or integer arrays,
          to avoid string parsing in reading applications.
+  \@AXISNAME_depends:
+    doc: |
+      Points to the path to a field defining the axis on which the ``AXISNAME`` axis depends.
```

Comment:
```suggestion
      Points to the path of a field defining the axis on which the ``AXISNAME`` axis depends.
```

Reply:
Added in new commit (so that nyaml and nxdl have the same text).